_posts/en/newsletters/2020-09-09-newsletter.md

```diff
@@ -99,9 +99,8 @@ deeper technical aspects.
       it sends an empty value for `nAbsurdFee`."
   a2link="https://bitcoincore.reviews/19339#l-85"
 
-  q3="Is `nAbsurdFee` part of policy, or is it a client preference?"
-
-  a3="The participants concluded that `nAbsurdfFee` is a client preference and
+  q3="Is `nAbsurdFee` part of policy, or is it a wallet preference?"
+  a3="The participants concluded that `nAbsurdfFee` is a wallet preference and
```

Comment:
Just to warn that in the meeting the participants consistently used the language "client preference" to describe their position regarding AbsurdFee.